message RequestRaftVoteResponse {
    required int32 term = 1;
    required bool voteGranted = 2;
}